spectree/plugins/flask_plugin.py

```diff
@@ -138,6 +148,7 @@ def validate(self,
 
         before(request, response, req_validation_error, None)
         if req_validation_error:
+            after(request, response, resp_validation_error, None)
```

Comment:
I guess it should be `after(request, response, req_validation_error, None)`

Reply:
yes you're right I think it should be `req_validation_error`